RequestService/RequestService.Repo/Repository.cs

```diff
@@ -1082,6 +1098,69 @@ public List<ShiftJob> GetUserShiftJobsByFilter(GetUserShiftJobsByFilterRequest r
                 jobs = jobs.Where(x => request.JobStatusRequest.JobStatuses.Contains((JobStatuses)x.JobStatusId)).ToList();
             };
 
+            if (request.DateFrom.HasValue)
+            {
+                jobs = jobs.Where(x => x.NewRequest.Shift.StartDate >= request.DateFrom.Value).ToList();
+            }
+
+            if (request.DateTo.HasValue)
+            {
+                jobs = jobs.Where(x => x.NewRequest.Shift.StartDate.AddMinutes(x.NewRequest.Shift.ShiftLength) <= request.DateTo.Value).ToList();
```

Comment:
Should we use the definition of EndDate built in to Shift here?  If we do something to solve for BST/GMT, ideally it'll be one change to take effect everywhere.